### DISCUSSION TITLE
Improve responsiveness of overall website

### DISCUSSION THREAD

**Discussion by TiBiBa** [hedyorg/hedy#3446] [Large Projects & Plans]
The website now is not really tailored towards smaller screens, but Tailwind has a lot of nice built-in responsiveness features. However, we don't use these as frequently as we should. Making the website less than perfect on smaller screens (tablets, phones for example). It would be a nice to improve the responsiveness of our site.

This discussion is meant to share ideas and approached that we might convert into smaller issues.

**Comment by Felienne** [comment]
Cleaning up discussions, this is more of an issue!
